# coveo_pcr_primers2021
### Identification of mutations in genomic regions directly targeted by publicly available PCR primers in SARS-CoV-2 samples

Due to the constantly increasing number of mutations in the SARS-CoV-2 genome, concerns have emerged over the possibility of decreased diagnostic accuracy of reverse transcription-polymerase chain reaction (RT-PCR), the gold standard diagnostic test for SARS-CoV-2. We propose an analysis pipeline to discover genomic variations overlapping the target regions of commonly used PCR primer sets. We provide the list of these mutations in a publicly available format based on a dataset of more than 1.2 million SARS-CoV-2 samples. Our approach distinguishes among mutations possibly having a damaging impact on PCR efficiency and ones anticipated to be neutral in this sense. Samples are categorized as „prone to misclassification” vs. „likely to be correctly detected” by a given PCR primer set based on the estimated effect of mutations present. Samples susceptible to misclassification are generally present at a daily rate of 2% or lower, although particular primer sets seem to have compromised performance when detecting Omicron samples. As different variant strains may temporarily gain dominance in the worldwide SARS-CoV-2 viral population, the efficiency of a particular PCR primer set may change over time, therefore constant monitoring of variations in primer target regions is highly recommended. 


VERSION_1: Raw data sets, pipelines, and figures containing the results of PCR analyses of SARS-CoV-2 mutations in SARS-CoV-2 target sites from the CoVEO database between 1 January and 31 December 2022.

VERSION_2: Raw data sets, pipelines, and figures containing results of PCR analyses of mutations in SARS-CoV-2 samples from the CoVEO database in SARS-CoV-2 target regions between 1 January and 6 April 2022.


Preprint: https://www.researchsquare.com/article/rs-1838361/v1
